### ISSUE TITLE
COG Analysis by month

### ISSUE THREAD

**Issue by AVHuntley** [afs1jes/Clarins-CA-IR-Migration#52]
Seems to run. Looks to me like a bunch of unformatted datasets that could be more useful as IR ad-hoc grids (remove on-sheet formatting, apply styles, use a slicer, etc)..

**Comment by FrederickGiroux** [comment]
Yeah i agree. This is not our best report. However, the ''Cost of purchase'' and ''gl 3521 only'' sheets are empty.

**Comment by afs1jes** [comment]
Corrected those sheets to use SelectYear instead of SystemCurrentYear. You should see data now.

**Comment by FrederickGiroux** [comment]
The sub at top and allocations sheet are empty. In the 2nd and 3rd sheet, there are red cells and the first column is too narrow

**Comment by A3Huntley** [comment]
I recopied sheets 'Sub at top' and 'Allocations' from the prod template. In prod, they're just ad hoc grids, not datasets, so they wouldn't necessarily get built with the model. In the new system, however, those grids will be treated the same as the others, just with fewer formatting options enabled.

Locked formulas in the formatting rows to prevent them displaying in red text within the grids.

I also copied the column widths from prod, so those should be identical now.
